ui/analyse/src/autoplay.js

```diff
@@ -28,7 +28,8 @@ module.exports = function(ctrl) {
       // in a variation
       if (!ctrl.tree.pathIsMainline(ctrl.vm.path)) return 2000;
       if (this.delay === 'realtime') {
-        return (ctrl.data.game.moveTimes[ctrl.vm.node.ply] * 100) || 2000;
+        if (ctrl.vm.node.ply < 2) return 2000;
```

Comment:
node.ply is the wrong measurement for this line and the one below.  Instead we need the index of this node, which might be ply or might be ply - 1 for black starts.